src/lib/Behat/Component/Table/TableRow.php

```diff
@@ -84,6 +92,7 @@ protected function specifyLocators(): array
             new VisibleCSSLocator('checkbox', 'input[type=checkbox]'),
             new VisibleCSSLocator('assign', '[data-original-title="Assign content"],[data-original-title="Assign to Users/Groups"]'),
             new VisibleCSSLocator('edit', '.ibexa-icon--edit,[data-original-title="Edit"]'),
+            new VisibleCSSLocator('copy', '.ibexa-icon--edit,[data-original-title="Copy"]'),
```

Comment:
```suggestion
            new VisibleCSSLocator('copy', '[data-original-title="Copy"]'),
```
The copy button does not use the edit icon, this should be removed